R/CI.R

```diff
@@ -210,14 +228,16 @@ nc_gety <- function(nc) {
 #' calculates daily climate anomalies from a given calibration period
 #' (default 1951-2005). These daily GCM anomalies are interpolated to
 #' the high-resolution observational grid. These interpolated daily
-#' anomalies constitute the "Climate Imprint". The high resolution
-#' gridded observations are then grouped into months and a climatology
+#' anomalies constitute the "Climate Imprint". If the high resolution
+#' gridded observations are daily observations instead of pre-computed
+#' monthly climatologies, they are then grouped into months and a climatology
 #' is calculated for each month. Finally the observed climatology is
 #' added to the GCM-based climate imprint and the final result is
 #' saved to output.file.
 #'
 #' @param gcm.file Filename of GCM simulations
-#' @param obs.file Filename of high-res gridded historical observations
+#' @param obs.file Filename of high-res gridded historical observations 
+#' (either daily data or monthly climatologies)
 #' @param output.file Filename to create (or overwrite) with the climate imprint outputs
 #' @param varname Name of the NetCDF variable to downscale (e.g. 'tasmax')
```

Comment:
This needs to be removed from the docstring, I think? Doesn't look like it accepts this variable anymore.